{
    "OBTResult": {
        "Env": {
            "BuildSystem": "Ubuntu_22_04_5_LTS",
            "Hardware": "CPU/Cores: 8, RAM: 23 GB",
            "IPAddress": "127.0.1.1 (OBT-BM01)"
        },
        "BuildSet": {
            "Branch": "master",
            "BranchDate": "Fri Jun 27 13:47:51 2025 +0100",
            "BranchSHA": "77D49ADE",
            "BuildType": "RelWithDebInfo",
            "Target": "BM/VM"
        },
        "Exclusion": {},
        "ThorConfig": {
            "Slaves": "4",
            "Channels": "1"
        },
        "Tasks": [
            {
                "Setup": {
                    "Result": "PASSED",
                    "thor": {
                        "Total": "3",
                        "Pass": "3",
                        "Fail": "0",
                        "Result": "PASSED"
                    },
                    "roxie": {
                        "Total": "5",
                        "Pass": "5",
                        "Fail": "0",
                        "Result": "PASSED"
                    }
                },
                "Performance": {
                    "Result": "FAILED",
                    "thor": {
                        "Total": "319",
                        "Pass": "316",
                        "Fail": "3",
                        "Result": "FAILED"
                    }
                }
            }
        ],
        "Errors": [
            {
                "thor": {
                    "02bl_sortvariable": {
                        "TestId": 80,
                        "Wuid": "W20250701-113158",
                        "Problem": "Exception",
                        "Code": "4",
                        "Msg": "System error: 4: WORKER #4 [192:168:1:59:20160]: 4, Unexpected process termination (ep:192:168:1:59:20160) : CSlavemsgHandler::threadmain"
                    },
                    "02bm_sortlarge": {
                        "TestId": 81,
                        "Wuid": "W20250701-113238",
                        "Problem": "Exception",
                        "Code": "4",
                        "Msg": "System error: 4: WORKER #2 [192:168:1:59:20120]: 4, Unexpected process termination (ep:192:168:1:59:20120) : CSlavemsgHandler::threadmain"
                    },
                    "02cd_multisort16": {
                        "TestId": 85,
                        "Wuid": "W20250701-113707",
                        "Problem": "Exception",
                        "Code": "5734",
                        "Msg": "System error: 5734: Graph graph1[1], sort[15]: Graph graph1[1], sort[15]: CSortMaster::ConnectSlaves: Could not connect to 192:168:1:59:20141, Master exception - caused by (5734, Graph graph1[1], sort[15]: CSortMaster::ConnectSlaves: Could not connect to 192:168:1:59:20141)"
                    }
                }
            }
        ]
    }
}